src/chrome/content/rules/FileFormat.info.xml

```diff
@@ -1,17 +1,21 @@
 <!--
+	403 Forbidden:
+		- samples.fileformat.info
 
-	Problematic domains:
+	MCB:
+		- blog.fileformat.info
 
-		fileformat.info		(connection refused)
+	Mixed content:
+		- blog.fileformat.info
 
+	Unable to connect:
+		- fileformat.info
+		- fonts.fileformat.info
 -->
-
-<ruleset name="FileFormat.info (MCB)" platform="mixedcontent">
-
-	<target host="fileformat.info" />
```

Comment:
Target and redirect to `www` should be kept, because `^` is still broken on https.

Reply:
Done.